src/opm/input/eclipse/Schedule/RSTConfig.cpp

```diff
@@ -163,7 +163,8 @@ bool is_RPTRST_mnemonic( const std::string& kw ) {
         "ACIP",     "ACIS",     "ALLPROPS", "BASIC",  "BG",       "BO",
         "BW",       "CELLINDX", "COMPRESS", "CONV",   "DEN",      "DENG",
         "DENO", "DENW", "DRAIN",
-        "DRAINAGE", "DYNREG",   "FIP",      "FLORES", "FLOWS",    "FREQ",
+        "DRAINAGE", "DYNREG",   "FIP",      "FLORES", "FLORES-", "FLOWS", 
+        "FLOWS-",   "FREQ",
```

Comment:
The sort order here is **potentially** ambiguous, in that `FLORES` might sort before or after `FLORES-` depending on the standard library implementation.  We might want to split our extension keywords out of the list and run a separate search for those if it becomes a problem.